libraries/AC_AttitudeControl/AC_AttitudeControl_Multi_6DoF.h

```diff
@@ -0,0 +1,95 @@
+#pragma once
+#ifdef ENABLE_SCRIPTING
+
+#include "AC_AttitudeControl_Multi.h"
+
+class AC_AttitudeControl_Multi_6DoF : public AC_AttitudeControl_Multi {
+public:
+    AC_AttitudeControl_Multi_6DoF(AP_AHRS_View &ahrs, const AP_Vehicle::MultiCopter &aparm, AP_MotorsMulticopter& motors, float dt):
+        AC_AttitudeControl_Multi(ahrs,aparm,motors,dt) {
+
+        if (_singleton != nullptr) {
+            AP_HAL::panic("Can only be one AC_AttitudeControl_Multi_6DoF");
+        }
+        _singleton = this;
+    }
+
+    virtual ~AC_AttitudeControl_Multi_6DoF() {}
+
+    static AC_AttitudeControl_Multi_6DoF *get_singleton() {
+        return _singleton;
+    }
+
+    // Command a Quaternion attitude with feedforward and smoothing
+    // not used anywhere in current code, panic so this implementaiton is not overlooked
+    void input_quaternion(Quaternion attitude_desired_quat) override {
```

Comment:
better to remove input_quaternion as not used

Reply:
removed function